### ISSUE TITLE
Make an Github organization for the Modus themes.

### ISSUE THREAD

**Issue by pacbru1260** [protesilaos/modus-themes#28]
Maybe we could try to make an organization for the amazing Modus themes and create various ports like for Terminator, Vim, Firefox themes and etc... kinda like what [this organization](https://github.com/catppuccin) is doing except I believe our organization should be 'simpler' in my opinion. What do you think of this, @protesilaos ?

**Comment by protesilaos** [comment]
Hello @pacbru1260!

> Maybe we could try to make an organization for the amazing Modus themes and create various ports like for Terminator, Vim, Firefox themes and etc... kinda like what [this organization](https://github.com/catppuccin) is doing except I believe our organization should be 'simpler' in my opinion. What do you think of this, @protesilaos?

I have thought about this before and believe the main problem is that we will give the wrong impression of what those ports are.  When an organisation is offering variants of what is supposed to be the same theme, users expect a consistent level of quality.  This, however, cannot be guaranteed due to the innate differences between the various applications that use those themes.  Also the fact that what we have here is not a pair of color schemes meaning that the color values are chosen to perform a given function, not look good together in abstract as part of a set---for example, the `blue-alt-other` is intentionally more saturated for `modus-operandi` due to how it is used and how a light background influences the perception of color, but I would not have necessarily made that choice if I were to design just a color scheme (use `M-x modus-themes-list-colors` to preview the palette).

I have two sections in the manual which explain why I think that is the case:

+ [Are these color schemes?](https://protesilaos.com/emacs/modus-themes#h:a956dbd3-8fd2-4f5d-8b01-5f881268cf2b)
+ [Port the Modus themes to other platforms?](https://protesilaos.com/emacs/modus-themes#h:7156b949-917d-488e-9a72-59f70d80729c)

To offer a concrete example of what I mean about the level of quality.  Have you ever used the `htop` program with a light theme that has high contrast?  It is virtually unusable because there are hardcoded colour combinations for the information bars with `termcol0` (black) text against a `termcol2` (green) background.  If you need a light theme with high contrast, check my [tempus-themes](https://github.com/protesilaos/tempus-themes) (this is a project that predates the modus-themes from which I learnt about the constraints with terminal emulators and the challenge with making ports).

In short:

1. ports will likely not have the quality of the Emacs implementation;
2. the organisation might give the impression that all ports are made equal.

What do you think?

If, by the way, you need a palette for your terminal emulator, the manual has samples for XTerm.  Check them out: <https://protesilaos.com/emacs/modus-themes#h:6b8211b0-d11b-4c00-9543-4685ec3b742f>.

**Comment by pacbru1260** [comment]
Alright, gotcha. I can understand why would you not want an organization for the Modus set. **_Maybe_** I could create an organization that has colorschemes that are based on the Modus themes (the credit would be ALL yours by the way)

**Comment by protesilaos** [comment]
> Alright, gotcha. I can understand why would you not want an
> organization for the Modus set.

I can always link to available ports in the manual.  That makes it
easier to comment on whether I have used them or not and state any other
proviso.

> **_Maybe_** I could create an organization that has colorschemes that
> are based on the Modus themes

Please go ahead.  This is free software, after all.  I am willing to
help you in any way I can.  For a quick-n-dirty exporter to get you
started, check this:
<https://github.com/protesilaos/dotfiles/blob/master/emacs/.emacs.d/prot-lisp/modus-themes-exporter.el>.

> (the credit would be ALL yours by the way)

There is no need.  Besides, you will be contributing as well.

**Comment by pacbru1260** [comment]
So I just finished making an organization and in case you wanna join, I've sent you the invite link to it @protesilaos github.com/modus-themes

**Comment by protesilaos** [comment]
Okay, I'm in!

What is your plan with this repo?  <https://github.com/modus-themes/modus-themes>.  Will it hold some generic files or do you want it to be a mirror of my repo?

* * *

Not related to this project, but my DElight shell script may come in handy for switching between light/dark themes across the environment: <https://github.com/protesilaos/dotfiles/blob/master/bin/bin/delight>.  At least it can be an inspiration for you, if you have some better ideas.

**Comment by protesilaos** [comment]
By the way, I don't have a logo for the themes.  What you got there is simply a small icon I was using in my GitLab profile to quickly identify my projects.  If you are good with design, feel welcome to come up with a logo.  Otherwise I can give it a try.

**Comment by protesilaos** [comment]
Again, feel welcome to design a logo.  Here are some drafts.

## The full "Modus themes" with >= 7:1 contrast for light and dark

![modus-themes-letters](https://user-images.githubusercontent.com/12828033/163155417-3bc95de9-a123-4986-9c54-31ca2e66c5ec.png)

## A bicoloured "M" with >= 7:1 contrast for light and dark

![modus-themes-m-bicolour](https://user-images.githubusercontent.com/12828033/163155422-e98d012e-a2cb-48fd-8ca3-4d842b125038.png)

## A unicoloured "M" with consistent ~4.5:1 contrast and more saturation

![modus-themes-m-unicolour](https://user-images.githubusercontent.com/12828033/163155424-0d100ce4-9b19-43c2-9b99-8e62ff90d977.png)

**Comment by pacbru1260** [comment]
Nice logo! When I get home, I'll update the logo in the repos

**Comment by pacbru1260** [comment]
> Okay, I'm in!
> 
> 
> 
> What is your plan with this repo?  <https://github.com/modus-themes/modus-themes>.  Will it hold some generic files or do you want it to be a mirror of my repo?
> 
> 
> 
> * * *
> 
> 
I plan for it to be a port collection of the modus themes 
> 
>

**Comment by protesilaos** [comment]
>> Okay, I'm in!
>> 
>> What is your plan with this repo?  <https://github.com/modus-themes/modus-themes>.  Will it hold some generic files or do you want it to be a mirror of my repo?
>
> I plan for it to be a port collection of the modus themes 

Very well!

**Comment by protesilaos** [comment]
Hello again!  I think we can safely close this issue now as the group has been created.

**Comment by pacbru1260** [comment]
> Hello again! I think we can safely close this issue now as the group has been created.

Yeah! I still consider the modus themes (especially vivendi) the best themes out there! Cheers :)

**Comment by protesilaos** [comment]
Thanks!
